CommonItem/src/test/java/com/github/sef24sp4/common/item/loottable/LootTableTest.java

```diff
@@ -0,0 +1,102 @@
+package com.github.sef24sp4.common.item.loottable;
+
+import com.github.sef24sp4.common.item.CommonItem;
+import com.github.sef24sp4.common.item.ItemRarity;
+import com.github.sef24sp4.common.item.ItemSPI;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class LootTableTest {
+	private LootTable lootTable;
```

Comment:
Remove instance field if it cannot generically be initialised in `setUp` method.
Currently its initialisation in `setUp` method is overridden by all tests. 
Consider if a generic initialisation method is possible. Otherwise use local variable instead.